zng/docs/zeek-compat.md

```diff
@@ -90,6 +92,17 @@ $ zq -t zeek_types.log | zq -f zeek -
 T	123	456	123.456	1592502151.123456	123.456	smile\xf0\x9f\x98\x81smile	\x09\x07\x04	80	127.0.0.1	10.0.0.0/8	tcp	a,in,set,things	order,is,important	Jeanne	122
 ```
 
+## About Aliases
+
+Multiple Zeek types discussed below are represented via a
+[type alias](spec.md#3117-alias-typedef) to one of ZNG's
+[primitive types](spec.md#5-primitive-types). The use of the alias maintains
+the history of the field's original Zeek type such that `zq` may restore it
+if/when the field may be later output again in Zeek format. Knowledge of its
+original Zeek type may also enable special operations in ZQL that are unique to
+values known to have originated as a specific Zeek type, though no such
+operations are currently implemented in `zq`.
+
```

Comment:
I think this would make a little more sense as a second paragraph in the next section (i.e., Type-Specific Details).

Reply:
Thanks for reviewing, @nwt. I took your suggestion of moving the paragraph and it looks good to me so I've committed it.